verkkokauppa/tests/test_merchant_requests.py

```diff
@@ -0,0 +1,216 @@
+from unittest.mock import Mock
+from uuid import UUID
+
+from assertpy import assert_that
+from django.conf import settings
+from pytest import raises
+from requests import Timeout
+
+from ..constants import REQUEST_TIMEOUT_SECONDS
+from ..merchants.exceptions import (
+    CreateMerchantError,
+    GetMerchantsError,
+    UpdateMerchantError,
+)
+from ..merchants.requests import (
+    create_merchant,
+    get_merchant,
+    get_merchants,
+    update_merchant,
+)
+from ..merchants.types import Merchant, MerchantInfo
+from .mocks import mock_get, mock_post
+
+# Create merchant
```

Comment:
If we used test classes we could make CreateMerchantTestCase and wouldn't need to comment :wink:

Reply:
I changed tests to TestClasses 👍